### ISSUE TITLE
launch stimulator but application open and close

### ISSUE THREAD

**Issue by junbinku** [infinitered/ignite#806]
The stimulator was launched on an IOS but when I click the application it opened and just closed automatically.

**Comment by jamonholmgren** [comment]
Hey @junbinku ! Thanks for using Ignite. It's totally fine to open an Ignite issue, but we usually need more information before we can determine whether it's an issue with Ignite or your app code.

For general help questions like this, I'd recommend you join our Slack community over at [community.infinite.red](http://community.infinite.red). You will find a friendly group of people over there that can give you feedback on your issue. If you determine it's an Ignite bug (or documentation, etc) then please do file another issue.
